lib/compiler/test/property_test/beam_types_prop.erl

```diff
@@ -217,7 +217,9 @@ gen_atom() ->
 gen_bs_matchable() ->
     oneof([?LET(Unit, range(1, 16), #t_bs_matchable{tail_unit=Unit}),
            ?LET(Unit, range(1, 16), #t_bs_context{tail_unit=Unit}),
-           ?LET(Unit, range(1, 16), #t_bitstring{size_unit=Unit})]).
+           ?LET(Unit, range(1, 16), #t_bitstring{size_unit=Unit}),
+           ?LET(Unit, range(1, 16), #t_bitstring{size_unit=Unit,
+                                                 appendable=true})]).
```

Comment:
```suggestion
           ?LET({Unit, Appendable}, {range(1, 16), boolean()},
                #t_bitstring{size_unit=Unit,appendable=Appendable})]).
```

Reply:
My Proper-fu failed me, that's nicer. Fixed.